manga_auto_translator/file_ops.py

```diff
@@ -14,7 +16,7 @@ def __init__(self, allowed_extensions: Sequence[str]=['.png', '.jpg', '.jpeg'])
     def load_scans(self, directory: str, img_mode: Union[None, str]='L') -> List[Scan]:
         scans = []
 
-        for file_name in os.listdir(directory)[:2]:
+        for file_name in os.listdir(directory):

```

Comment:
FYI (optional) : Recently I abused pahtlib, it's better to manage arborescence and files. It allow you to play directly with a path object and not use os library.